src/components/Header.js

```diff
@@ -1,9 +1,13 @@
 import React from 'react';
+import icon from '../assets/icon3.png';
```

Comment:
Hey, so what do you think if we change this icon?

Reply:
Of course! We have 3 other options there in `assets`, which one would you say we should use?

Reply:
We don't have to change it now! Next week maybe? :)